deploy/openshift/templates/che-server-template.yaml

```diff
@@ -153,6 +153,8 @@ objects:
             value: "${CHE_WORKSPACE_PLUGIN__REGISTRY__URL}"
           - name: CHE_TRACING_ENABLED
             value: "${CHE_TRACING_ENABLED}"
+          - name: CHE_WSMASTER_CORS_FILTER_ENABLED
```

Comment:
hello, question: why do we need 'ENABLED' in the property name ?
if values are true/false, we do know that if it's true it means it's enabled, and if false it is disabled (like any boolean properties)

Reply:
I think `CHE_WSMASTER_CORS_FILTER_ENABLED` describes format of its value, while you can not really guess what it a value of `CHE_WSMASTER_CORS_FILTER`: `true/false`, `enabled/disabled`, `CheCorsFilterImpl/DummyFilterImpl`. @benoitf Do you find it meaningful?
So `ENABLED` in the property name looks good to me.

Reply:
@sleshchenko well then it would mean that
`CHE_MULTIUSER` should be `CHE_MULTIUSER_ENABLED`, `TLS` should be `TLS_ENABLED`, etc ?

you may also just replaced FILTER by FILTERED if it's more meaningful: CHE_WSMASTER_CORS_FILTERED

Reply:
> CHE_MULTIUSER should be CHE_MULTIUSER_ENABLED, TLS should be TLS_ENABLED, etc ?

Why not? =)

> `CHE_WSMASTER_CORS_FILTERED` 

I like the idea but I'm not really sure that we should apply it.
In case of `che.docker.privileged` it sounds good. Docker is in privileged mode.
In this case, I may be wrong but for me, CORS_FILTER__ENABLED sounds cors filter is enabled
`CORS_FILTERED` cors is filtered?

Reply:
For me `CHE_CORS` (true/false) is enough :)